src/data.rs

```diff
@@ -69,7 +70,7 @@ pub enum SimpleType<'a> {
 	/// Tuple struct or tuple variant.
 	Tuple(&'a Fields<'a>),
 	/// Union.
-	Union(&'a Fields<'a>),
+	Union(#[allow(unused)] &'a Fields<'a>),
```

Comment:
If its unused, can we remove it?
Or whats going on here?

(sorry, didn't check myself yet)

Reply:
yeah, not sure why we even have it in this enum, probably because we are parsing it?

Reply:
Yep, removed!